apps/ingest-from-bucket/app/bucket_loader.py

```diff
@@ -18,16 +18,21 @@ def main(args):
     db = create_connector()
 
     if args.delete:
-        WorkflowSpec.delete( "bucket-loader", args.spec_id )
+        logger.info(f"Deleting all information for spec {args.spec_id}")
+        WorkflowSpec.delete_spec(db, "bucket-loader", args.spec_id )
+        print(f"Completed delete all bucket loader for spec {args.spec_id}")
         sys.exit(1)
     elif args.delete_all:
-        WorkflowSpec.delete_all( "bucket-loader")
+        WorkflowSpec.delete_all_data(db, "bucket-loader")
+        print("Completed delete all bucket loader data")
+        sys.exit(1)
+    elif args.clean_bucket:
+        WorkflowSpec.clean_bucket(db, args.cloud_provider+":",args.bucket )
+        print("Completed bucket clean action")
         sys.exit(1)
 
 
     spec = WorkflowSpec( db, "bucket-loader", args.spec_id, clean=args.clean )
-    #if args.clean:
-    #    clean_data( generate_spec_key(args.cloud_provider,args.bucket))
 
     run_id = uuid4()
```

Comment:
```suggestion
    run_id = str(uuid4())
```

Reply:
I just went and made it str and removed where it was changed to str in everywhere else